implementations/rust/ockam/ockam_entity/src/entity.rs

```diff
@@ -105,7 +105,7 @@ impl Entity {
 impl ProfileAdd for Entity {
     fn add_profile(&mut self, profile: ProfileSync) -> Result<()> {
         if let Ok(id) = profile.identifier() {
-            if let Some(_) = self.profiles.insert(id, profile) {
+            if self.profiles.insert(id, profile).is_some() {
```

Comment:
@jared-s does it conflict with your changes?

Reply:
oh yes :-) my changeset around this is huge. i will need to take the merge.